### PR TITLE
Add ability to vary number of cores stressed

### PR DESCRIPTION
**Pull request by shortbloke**
Add the ability to run 'stress' on a subset of cores via command-line argument.
Change default test to use all cores, rather than hardcoded to 4. (Pi Zero only has a single core).

### CODE REVIEW

File: stressberry/main.py [hunk 1]
```diff
@@ -35,10 +36,28 @@ def measure_temp(filename="/sys/class/thermal/thermal_zone0/temp"):
     return temp
 
 
-def test(duration):
-    print("Idling...")
-    tme.sleep(0.25 * duration)
-    stress_cpu(4, time=0.5 * duration)
-    print("Idling...")
-    tme.sleep(0.25 * duration)
+def cpu_core_count():
```
Comment (nschloe) [inline stressberry/main.py]: No need to wrap this.

Comment (shortbloke) [reply stressberry/main.py]: Fixed

File: stressberry/main.py [hunk 2]
```diff
@@ -35,10 +36,21 @@ def measure_temp(filename="/sys/class/thermal/thermal_zone0/temp"):
...
+def test(duration, cores):
+    """Run stress test with 25% of test duration for idle before and after the stres
+    """
+    stress_duration = 0.5 * duration
+    idle_duration = 0.25 * duration
+
+    if cores is None:
+        cores = cpu_count()
+
+    print("Preparing to stress: [{}] CPU Cores for [{}] seconds".format(cores, stress_duration))
```
Comment (nschloe) [inline stressberry/main.py]: Typo? Remove the `:`.

**Comment by nschloe** [comment]
Looks good to me now! Not sure why the tests aren't catching on, I'll look at this later.

**Comment by nschloe** [comment]
circleci was misconfigured -- could you add a dummy commit to see if it works now?
```
git commit --allow-empty -m "Trigger tests"
```

**Comment by nschloe** [comment]
Thanks!

**Pull request merged**